src/components/FormValidator.js

```diff
@@ -65,6 +65,11 @@ class FormValidator {
     });
     this._setEventListeners();
   }
+
+  reset() {
```

Comment:
* `reset` should be named `disableButton` because you disable the button here


```js

  disableButton() {
    this._submitButton.classList.add(this._inactiveButtonClass);
    this._submitButton.disabled = true;
  }
```